stories/carousel/carousel.stories.js

```diff
@@ -0,0 +1,480 @@
+                <li 
+                    data-slide-to="2" 
+                    class="fd-carousel__page-indicator"></li>
+                <li 
+                    data-slide-to="3" 
+                    class="fd-carousel__page-indicator"></li>
+                <li 
+                    data-slide-to="4" 
+                    class="fd-carousel__page-indicator"></li>
+                <li 
+                    data-slide-to="5" 
+                    class="fd-carousel__page-indicator fd-carousel__page-indicator--active"></li>
+                <li 
+                    data-slide-to="6" 
+                    class="fd-carousel__page-indicator"></li>
+                <li 
+                    data-slide-to="7" 
+                    class="fd-carousel__page-indicator"></li>
+            </ol>
+        </div>
+    </div>
+    <div style="display: none;" role="region" id="carousel-2" aria-live="polite">
+        Displaying item 1 of 4
+    </div>
+
+    <h4>Numeral Format for Page Indicator</h4>
+    <div 
+        class="fd-carousel" 
+        data-ride="carousel" 
+        style="margin-bottom: 3rem; max-width: 30rem;">
+        <div class="fd-carousel__content" style="min-height: 15.5rem;"></div>
+        <div class="fd-carousel__page-indicator-container">
+            <button 
+                class="fd-button fd-carousel__button sap-icon--slim-arrow-left"
+                data-slide="prev"
+                aria-label="Go to previous item">
+            </button>
+            <div class="fd-carousel__page-indicators">
+                <div class="fd-carousel__text">1 of 4</div>
+            </div>
+            <button 
+                class="fd-button fd-carousel__button sap-icon--slim-arrow-right"
+                data-slide="next"
+                aria-label="Go to next item">
+            </button>
+        </div>
+    </div>
+    <div style="display: none;" role="region" id="carousel-3" aria-live="polite">
+        Displaying item 1 of 4
+    </div>
+
+    <h4>No Page Indicator</h4>
+    <div 
+        class="fd-carousel" 
+        data-ride="carousel" 
+        style="margin-bottom: 3rem; max-width: 30rem;">
+        <div class="fd-carousel__content" style="min-height: 15.5rem;">
+            <button 
+                class="fd-button fd-carousel__button sap-icon--slim-arrow-left"
+                data-slide="prev"
+                aria-label="Go to previous item">
+            </button>
+            <button 
+                class="fd-button fd-carousel__button sap-icon--slim-arrow-right"
+                data-slide="next"
+                aria-label="Go to next item">
+            </button>
+        </div>
+    </div>
+    <div style="display: none;" role="region" id="carousel-4" aria-live="polite">
+        Displaying item 1 of 4
+    </div>
+</div>
+`;
+
+carouselBottom.storyName = 'Carousel with Page Indicator on Bottom';
+carouselBottom.parameters = {
+    docs: {
+        iframeHeight: 900,
+        storyDescription: `
```

Comment:
"The bottom page indicator carousel displays a content area, navigation buttons and a page indicator at the bottom. The page indicator wraps onto the next line if the content is too long, and after the pages reach a count of 8, the dots are replaced by numerical representation i.e. _1 of 9_."